hr_contract_amendment/models/hr_contract.py

```diff
@@ -0,0 +1,92 @@
+# -*- coding: utf-8 -*-
```

Comment:
Use a different file for contract and amendment, split it at least in 2

Reply:
ok  it is done 
Thanks for reviews !

Reply:
made